docs/slack-apps/tutorials/channel-check.md

```diff
@@ -0,0 +1,70 @@
+---
+id: channel-check
+title: Check for New Channels
+sidebar_label: Check for New Channels
+---
+
+It can be hard to keep up with all of the changes in your Slack workspace! This tutorial will walk you through how to build a simple tool in Dark that posts all new, public Slack channels created in your Slack workspace in the past 24 hours. 
+
+A full sample canvas for this tutorial is available [here](https://darklang.com/a/sample-slackchannelcheck).
+
+## Slack Configuration
+
+1. [Create a new app in Slack](https://api.slack.com/apps).
+2. Navigate to Features -> OAuth& Permissions, scroll down to scopes and add the following bot token scopes: `channels:manage`, `channels:read`, `chat:write`, and `chat:write.public`. This will allow our app to interact with our the channels (also known as conversations) and post messages about them. 
+
+![assets/slack-apps/tutorials/channelcheck/bottokenscopes.png](assets/slack-apps/tutorials/channelcheck/bottokenscopes.png)
+
+3. Install your app to the workspace (either via Settings -> Basic Information or Settings -> Install App). Since this app is being built for your workspace specifically, we will not need to set up full OAuth, like we would if we wanted to distribute this more publicly.
+4. Copy your bot user access token, which will now be available under Settings -> Install app and save it as a function in your Dark canvas. Use this function wherever you see `token` in the code examples.
+5. If no new channels have been created in your Slack recently, create a new public channel so we have some data to work with.
+6. You will need to choose a Slack channel to post your messages to. You can find your Slack channel's id by navigating to the channel in a browser and looking for the 11 character string that begins with a capital C - it will look something like `C1234567891`. Add that to its own Dark function as well, and use it wherever you see `channel` in the code examples.
+
+## Building in Dark
+
+1. Create a new daily CRON named checkForNewChannels by clicking the + in the sidebar next to Cron, and call the [getNewConversations Slack function](../slack-packages#getnewconversations) (it's a little confusing, but channels are referred to as conversations in the Slack API). The getNewConversations function is built into Dark's [package manager](./../../packages), and it takes your token as well as an amount of time to check in Epoch seconds. Here, we're asking it to check in the past 24 hours, which is equivalent to 86400 seconds.
```

Comment:
```suggestion
1. Create a new daily CRON named `checkForNewChannels` by clicking the `+` in the sidebar next to `Cron`, and call the [getNewConversations Slack function](../slack-packages#getnewconversations) (it's a little confusing, but channels are referred to as conversations in the Slack API). The `getNewConversations` function is built into Dark's [package manager](./../../packages), and it takes your token as well as an amount of time to check in Epoch seconds. Here, we're asking it to check in the past 24 hours, which is equivalent to 86400 seconds.
```

Reply:
Can you use code blocks for all the functions names and types on this page?

Reply:
CRON is spelled `Cron` (there's a few other places in this doc with this spelling)